fastapi_cache/decorator.py

```diff
@@ -81,7 +82,23 @@ def _uncacheable(request: Optional[Request]) -> bool:
         return False
     if request.method != "GET":
         return True
-    return request.headers.get("Cache-Control") == "no-store"
+    return False
+
+def _extract_cache_control_headers(request: Optional[Request]) -> Set[str]:
+    """Extracts Cache-Control header
+    1. Convert all header to lowercase to make it case insensitive
+    2. Split on comma (,)
+    3. Strip whitespaces
+    4. convert to all lower case
+
+    returns an empty set if header not set
+    """
+    if request is not None:
+        headers = {header_key.lower(): header_val for header_key, header_val in request.headers.items()}
+        cache_control_header = headers.get("cache-control", None)
+        if cache_control_header:
+            return {cache_control_val.strip().lower() for cache_control_val in cache_control_header.split(",")}
```

Comment:
And here as well.

Reply:
sorry didn't get this one. This is already a set comprehension.

Reply:
this is case conversion of the header value which Fastapi doesn't convert based on some of my testing

Reply:
Yeah, sorry for confusion, I mean header keys are case insensitive.